ansible/azure-aap-deployment-driver
# AAP 10179 - Segment publisher from Installer

# Description

This PR is for creating a Segment client in Installer that will publish all data to Segment. The publisher is generic and will try to consume all the metrics available from the Telemetries table. If not available, it will pass an empty string to Segment.




## PR task checklist



# Ticket

https://issues.redhat.com/browse/AAP-10179


# Testing

1. Pull down the PR.
2. Create a new deployment using the installerincluded branch of aap-azurerm repo.
3. Once the Deployment is complete, go to Segment (https://app.segment.com/redhat-devtools/sources/ansible_on_azure_managed_app_dev/debugger) and check if the event is published here.
